index.html

```diff
@@ -237,7 +237,7 @@ <h3>
             <dd>
               <p>
 This family of specifications consists of documents that each define how to
-express proofs of integrity for verifiable credentials using a number of
+express and associate proofs of integrity with data, focusing on verifiable credentials and
 concrete serializations for each of the defined syntaxes. The specific set of
```

Comment:
A slight tweak:

```suggestion
express and associate proofs of integrity for verifiable credentials and
concrete serializations for each of the defined syntaxes. The Working Group
would welcome the usage of these techniques for data in general, but
its scope will be to solve verifiable credentials use cases. The specific set of
```

Reply:
+1 to this particular suggestion. @Sakurann and @iherman -- thoughts?

Reply:
Looks good..

Reply:
LGTM. Thank you, Dave.